templates/genie.yaml

```diff
@@ -82,6 +106,19 @@ Resources:
           Order: 1
       Priority: 1
       State: ENABLED
+  DevBatchJobQueue:
+    Type: "AWS::Batch::JobQueue"
+    Properties:
+      JobQueueName: !Join
```

Comment:
This is the same name as `BatchJobQueue`, I think these need to be unique.

Reply:
Ah, didn't notice these.  I thought all I needed to change was the bottom
```
 BatchJobQueue:
    Value: !Ref BatchJobQueue
    Export:
      Name: !Sub '${AWS::Region}-${AWS::StackName}-BatchJobQueue'
  DevBatchJobQueue:
    Value: !Ref DevBatchJobQueue
    Export:
      Name: !Sub '${AWS::Region}-${AWS::StackName}-DevBatchJobQueue'
```